### PR TITLE
Our timeout was 5min

### PR DESCRIPTION
**Pull request by TheOneRing**

### CODE REVIEW

**Comment by ownclouders** [comment]
Results for <strong>GUI-tests</strong> https://drone.owncloud.com/owncloud/client/10812/5/1
:boom: The GUI tests failed.
GUI Logs: (https://cache.owncloud.com/public/owncloud/client/10812/guiReportUpload/index.html)            
Server Logs: ((https://cache.owncloud.com/public/owncloud/client/10812/guiReportUpload/serverlog.log)

**Comment by ownclouders** [comment]
Results for <strong>GUI-tests</strong> https://drone.owncloud.com/owncloud/client/10814/5/1
:boom: The GUI tests failed.
GUI Logs: (https://cache.owncloud.com/public/owncloud/client/10814/guiReportUpload/index.html)            
Server Logs: ((https://cache.owncloud.com/public/owncloud/client/10814/guiReportUpload/serverlog.log)

File: src/libsync/abstractnetworkjob.h [hunk 1]
```diff
@@ -107,6 +107,12 @@ class OWNCLOUDSYNC_EXPORT AbstractNetworkJob : public QObject
      */
     static std::chrono::seconds httpTimeout;
 
+    /**
+     * The default 5 minutes timeout if none is specified by the config.
+     * Qt's default would be 30s.
+     */
+    static const std::chrono::seconds DefaultHttpTimeout;
```
Comment (fmoc) [inline src/libsync/abstractnetworkjob.h:114]: Can be `constexpr`, too.

**Comment by ownclouders** [comment]
Results for <strong>GUI-tests</strong> https://drone.owncloud.com/owncloud/client/10815/5/1
:boom: The GUI tests failed.
GUI Logs: (https://cache.owncloud.com/public/owncloud/client/10815/guiReportUpload/index.html)            
Server Logs: ((https://cache.owncloud.com/public/owncloud/client/10815/guiReportUpload/serverlog.log)

**Review by fmoc — APPROVED** [review]

**Comment by ownclouders** [comment]
Results for <strong>GUI-tests</strong> https://drone.owncloud.com/owncloud/client/10830/5/1
:boom: The GUI tests failed.
GUI Logs: (https://cache.owncloud.com/public/owncloud/client/10830/guiReportUpload/index.html)            
Server Logs: ((https://cache.owncloud.com/public/owncloud/client/10830/guiReportUpload/serverlog.log)

**Pull request merged**